src/components/Game.js

```diff
@@ -13,8 +13,13 @@ const gamePadding = 8;
 const cardArray = generateCards();
 
 function Game({ deck, onClick, onClear, selected }) {
-  const [layout, setLayout] = useStorage("layout", "vertical");
```

Comment:
Could we not change the name of the existing `"layout"` key in storage, if possible?

Reply:
My bad, I didn't think of the consequences for existing users.